ngraph/test/attributes.cpp

```diff
@@ -1104,15 +1104,26 @@ TEST(attributes, lstm_cell_op)
 TEST(attributes, lstm_sequence_op)
 {
     FactoryRegistry<Node>::get().register_factory<opset1::LSTMSequence>();
-    const auto X = make_shared<op::Parameter>(element::f32, Shape{1, 2, 4});
-    const auto initial_hidden_state = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3});
-    const auto initial_cell_state = make_shared<op::Parameter>(element::f32, Shape{1, 2, 3});
-    const auto sequence_lengths = make_shared<op::Parameter>(element::i32, Shape{2});
-    const auto W = make_shared<op::Parameter>(element::f32, Shape{1, 12, 4});
-    const auto R = make_shared<op::Parameter>(element::f32, Shape{1, 12, 3});
-    const auto B = make_shared<op::Parameter>(element::f32, Shape{1, 12});
 
+    const auto batch_size = 2;
+    const auto num_directions = 1;
+    const auto seq_length = 1;
```

Comment:
Can we have a non-trivial value for `num_directions` and `seq_length`? I mean the value not equal to 1. Also, it will be great if all values in lines 1108-1112 are different.

Reply:
Sure,  done